LobbyServer2/LobbyServer/Session/SessionTicketData.cs

```diff
@@ -0,0 +1,65 @@
+﻿using EvoS.Framework;
+using Org.BouncyCastle.Asn1.Mozilla;
+using Org.BouncyCastle.Crypto.Parameters;
+using System;
+using System.Collections.Generic;
+using System.Drawing.Text;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace CentralServer.LobbyServer.Session
+{
+    [Serializable]
+    public class SessionTicketData
+    {
+        public long AccountID;
+        public long SessionToken;
+        public long ReconnectionSessionToken;
+        private static readonly HashAlgorithm algorithm = SHA256.Create();
+        private static readonly Guid saltGuid = Guid.NewGuid(); // new salt everytime the server restarts
+
+        public string ToString()
+        {
+            return $"{AccountID}\n{SessionToken}\n{ReconnectionSessionToken}";
+        }
+
+        public string ToStringWithSignature()
+        {
+            return GetSignature()+"\n"+this.ToString();
+        }
+
+        public static SessionTicketData FromString(string data)
+        {
+            string[] parts = data.Split("\n");
+            if (parts.Length < 3) return null;
```

Comment:
< 4?

Reply:
i changed it to !=4, before i had less fields and it got outdated